src/Job/Executor.php

```diff
@@ -238,7 +236,12 @@ public function execute(Job $job)
                 }
             }
 
-            $this->overrideComponentImageTag($params, $componentConfiguration, $componentClass);
+            $componentClass->setImageTag(TagResolverHelper::resolveComponentImageTag(
+                $params,
+                $componentConfiguration,
+                $componentClass
+            ));
+            $this->logger->info(sprintf('Using component tag: "%s"', $componentClass->getImageTag()));
```

Comment:
:smile: koukam ze jsem se az moc drzel toho, jak to bylo - ze se loguje override. Krasny elegantni reseni :heart:

Reply:
zavazelo to :)